src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header-img-link">
+        <img src="./images/logo.png"
+             alt="MOYO logotype"
+             class="header-img">
```

Comment:
Nice alt, check out the code style tips [here](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md). Attibutes should be on separate lines (if the tag is multiline).
```suggestion
        <img 
          src="./images/logo.png"
          alt="MOYO logotype"
          class="header-img"
        >
```